middlebury.edu/2010/index.html

```diff
@@ -3,16 +3,17 @@
   <head>
     <title>Middlebury</title>
     <meta charset="utf-8">
-    <link rel="stylesheet" href="//cdn.middlebury.edu/middlebury.edu/2010/css/midd.css">
-    <link rel="stylesheet" href="//cdn.middlebury.edu/middlebury.edu/2010/css/print.css" media="print" />
-    <link rel="shortcut icon" href="//cdn.middlebury.edu/middlebury.edu/2010/images/midd_favicon.ico">
-    <link rel="apple-touch-icon" href="//cdn.middlebury.edu/middlebury.edu/2010/images/apple-touch-icon/icon.png">
-    <link rel="apple-touch-icon" sizes="76x76" href="//cdn.middlebury.edu/middlebury.edu/2010/images/apple-touch-icon/icon-ipad.png">
-    <link rel="apple-touch-icon" sizes="120x120" href="//cdn.middlebury.edu/middlebury.edu/2010/images/apple-touch-icon/icon-retina.png">
-    <link rel="apple-touch-icon" sizes="152x152" href="//cdn.middlebury.edu/middlebury.edu/2010/images/apple-touch-icon/icon-ipad-retina.png">
+    <link rel="stylesheet" href="//cloud.typography.com/83898/706148/css/fonts.css">
+    <link rel="stylesheet" href="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/css/midd.css">
+    <link rel="stylesheet" href="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/css/print.css" media="print" />
+    <link rel="shortcut icon" href="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/images/midd_favicon.ico">
+    <link rel="apple-touch-icon" href="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/images/apple-touch-icon/icon.png">
+    <link rel="apple-touch-icon" sizes="76x76" href="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/images/apple-touch-icon/icon-ipad.png">
+    <link rel="apple-touch-icon" sizes="120x120" href="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/images/apple-touch-icon/icon-retina.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/images/apple-touch-icon/icon-ipad-retina.png">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
-    <script src="//cdn.middlebury.edu/middlebury.edu/2010/js/midd.js"></script>
-    <script type="text/javascript" src="//cdn.middlebury.edu/middlebury.edu/2010/js/modernizr.min.js"></script>
+    <script src="//hammer.middlebury.edu/~imcbride/cdn/middlebury.edu/2010/js/midd.js"></script>
```

Comment:
Should this still point to cdn?

Reply:
Good catch. I'll get that fixed. Just so you know this is an old, low-priority issue that I'm migrating from Kanbanize into Github issues.

Reply:
I just was looking at github notifications so thought I'd leave the comment for now while I saw it. I won't get to an actual review for awhile I'm sure.